src/pretix/control/templates/pretixcontrol/event/invoicing.html

```diff
@@ -59,6 +60,99 @@ <h1>{% trans "Invoice settings" %}</h1>
                 {% bootstrap_field form.invoice_renderer_highlight_order_code layout="control" %}
                 {% bootstrap_field form.invoice_eu_currencies layout="control" %}
             </fieldset>
+            <fieldset>
+                <legend>{% trans "Invoice transmission" %}</legend>
+                <p class="help-block">
```

Comment:
```suggestion
                <p>
```
The gray text color makes it a bit hard to read